solver/llbsolver/solver.go

```diff
@@ -390,13 +395,14 @@ func oneOffProgress(ctx context.Context, id string) func(err error) error {
 	}
 }
 
-func inBuilderContext(ctx context.Context, b solver.Builder, name, id string, f func(ctx context.Context, g session.Group) error) error {
+func inBuilderContext(ctx context.Context, b solver.Builder, name, id string, pg *pb.ProgressGroup, f func(ctx context.Context, g session.Group) error) error {
```

Comment:
What's the case where progressgroup is needed here? Isn't this function by definition for creating a separate progress row?

Reply:
I don't understand. How is this different from merge with no progressgroup? ` e.Name()` is the exporter name in here and should always be shown and never combined with any other vertex.

Reply:
I went back to disable this to show what the progress looks like without it and it looks totally fine without it now 😞 At one point unlazies that occurred during an export were not showing up in the progress group and I thought this was needed to fix it, but I must have fixed it some other way. I will confirm tomorrow morning and get rid of this if not needed.

Reply:
Yeah, what happened was I fixed the export issue with this change originally, but then another fix I made at a later time inadvertently also served as a much simpler fix to it. So I updated with all this extra complication removed thankfully.